rustworkx/rustworkx.pyi

```diff
@@ -50,6 +50,12 @@ class FailedToConverge(Exception): ...
 class InvalidMapping(Exception): ...
 class GraphNotBipartite(Exception): ...
 
+@final
+class GreedyStrategy:
```

Comment:
I would probably change this to `ColoringStrategy` or `GreedyColoringStrategy` to be a bit more descriptive.